packages/effects-runtime/test/__fixtures__/functional-tests/errors-in-handlers.js

```diff
@@ -38,16 +32,59 @@ const asyncEjectCase = async () => {
     }
 };
 
-const continuationEjectCase = () => {
+const catchingEject = async () => {
+    'use effects';
+    try{
+
+        try{
+            try{
+                perform {type : 'innerPerform'}
+            }handle default with (e){
+                syncEject();
+            }
+
+        }catch(e){
+            console.log('catching');
+            return perform {type : 'outerPerform'}
+        }
+
+
+    }handle default with (e){
+        recall 'outer perform!';
+    }
+};
+
+const bubblingEffect = async () => {
   'use effects';
   try{
 
-  }handle ejectType with (e){
-      continuationEject();
+      try{
+
+          try{
+
+              try{
+
+                  try{
```

Comment:
biiig giiirls dooont try, they dont try yi yi yi--_they dont try (burly voice)_

Reply:
As ridiculous as this test may look, I'm super happy this works now